src/pages/billing/CreateNewBillingProjectWizard.js

```diff
@@ -0,0 +1,367 @@
+    } catch (error) {
+      if (error.status === 409) {
+        setExisting(_.concat(billingProjectName, existing))
+      } else {
+        throw error
+      }
+    }
+  })
+
+  const step1 = () => {
+    const isActive = activeStep === 1
+    const isDone = activeStep > 1
+
+    return li({ 'aria-current': isActive ? 'step' : false, style: { ...styles.stepBanner(isActive), flexDirection: 'row', alignItems: 'center', justifyContent: 'space-between' } }, [
+      span({ className: 'sr-only' }, ['Step 1, step 1 of 4']),
+      div({ style: { width: '60%' } }, [
+        h2({ style: { fontSize: 18, marginTop: 0 } }, ['STEP 1']),
+        span({ style: { fontSize: 14, lineHeight: '22px', whiteSpace: 'pre-wrap' } },
+          ['Go to the Google Cloud Platform Billing Console and sign-in with the same user you use to login to Terra.'])
+      ]),
+      h(Clickable, {
+        style: {
+          color: '#426ead', backgroundColor: 'none', border: '1px solid #426ead',
+          paddingInline: '1.5rem', display: 'inline-flex', justifyContent: 'space-around', alignItems: 'center',
+          height: '2.5rem', fontWeight: 500, fontSize: 14, borderRadius: 2, whiteSpace: 'nowrap',
+          marginLeft: '2rem', textTransform: 'none'
+        },
+        href: 'https://console.cloud.google.com',
+        ...Utils.newTabLinkProps,
+        onClick: () => {
+          if (!isDone) {
+            next()
+          }
+        }
+      }, ['Go to Google Cloud Console'])
+    ])
+  }
+
+  const step2 = () => {
+    const isActive = activeStep === 2
+    const isDone = activeStep > 2
+
+    const setNextStep = () => {
+      if (activeStep === 1) {
+        setActiveStep(3)
+      } else if (isDone) {
+        setActiveStep(3)
+        setAccessToAddBillingAccountUser(undefined)
+        setVerified(false)
+      } else {
+        next()
+      }
+    }
+
+    return li({ 'aria-current': isActive ? 'step' : false, style: { ...styles.stepBanner(isActive), display: 'flex', flexDirection: 'column' } }, [
+      span({ className: 'sr-only' }, ['Step 2, step 2 of 4']),
+      h2({ style: { width: '55%', fontSize: 18, marginTop: 0 } }, ['STEP 2']),
+      fieldset({ style: { border: 'none', margin: 0, padding: 0, display: 'block' } }, [
+        div({ style: { width: '55%', fontSize: 14, lineHeight: '22px', whiteSpace: 'pre-wrap', marginTop: '0.25rem', float: 'left' } },
+          ['Select an existing billing account or create a new one.\n\nIf you are creating a new billing account, you may be eligible for $300 in free credits. ' +
+          'Follow the instructions to activate your account in the Google Cloud Console.']),
+        div({ style: { width: '25%', float: 'right' } }, [
+          div({ style: { display: 'flex', displayDirection: 'row' } }, [
+            h(RadioButton, {
+              text: 'I don\'t have access to a Cloud billing account', name: 'access-to-account',
+              'aria-checked': accessToBillingAccount === false,
```

Comment:
This was suggested by @gabedemattos, but it would be best to delete the `aria-checked` statements. Because we are using semantic HTML (inputs of type radio button), this is handled natively for us and there is no need to duplicate the information.

Here is what the screen reader will announce without `aria-checked` present:

<img width="636" alt="image" src="https://user-images.githubusercontent.com/484484/198095510-a42468fa-8307-4a96-ab3a-68783b20d9a3.png">